<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiEditFormatDialog</class>
 <widget class="QDialog" name="MultiEditFormatDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>411</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Format name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="FormatNameEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="4,0,5">
     <property name="topMargin">
      <number>12</number>
     </property>
     <property name="bottomMargin">
      <number>12</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,1">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Header:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="HeaderComboBox"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Columns separator:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ColSepEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="DateGroupBox">
         <property name="title">
          <string>Date format</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,1">
            <item>
             <widget class="QRadioButton" name="PredefDateRadioButton">
              <property name="text">
               <string>Predefined:</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="DateComboBox"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QRadioButton" name="CustomDateRadioButton">
              <property name="text">
               <string>Custom:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="CustomDateEdit"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Precision:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="PrecisionSpinBox">
           <property name="value">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Comment character:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="CommentCharEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Missing value string:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="MissingValueEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::VLine</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Preview:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="PreviewEdit">
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWrapMode">
          <enum>QTextEdit::NoWrap</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="ButtonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
